tests/helpers.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from random import choice
+from string import ascii_letters
+from time import time
+
+from six.moves import xrange
+
+
+def randstr(n=5):
+    """Generates a random string with n ascii_letters"""
+    return ''.join(choice(ascii_letters) for _ in xrange(n))
+
+
+def uniqstr():
```

Comment:
Aren't there any functions in Python's standard module `uuid` that could be used? Just wondering.

Reply:
UUID could be used, but some extra logic might be required, because I believe python logging required valid identifiers (something like stripping dashes and prepending letters).

If you prefer, I can go in that direction.

Reply:
No, I was just wondering if it was considered. I always tend to prefer simpler solution. So if using `uuid` would result in something even simpler but I guess we should keep it as it is right now.

Reply:
I did some experiments, and it seems that, despite [suggesting it](https://docs.python.org/3.6/library/logging.html#logger-objects), the `module identifier` format is just recommended, not required.

I will try to remove `uniqstr` in favor of `uuid4` and see what happens :)

Thanks for the suggestion.